master/buildbot/util/service.py

```diff
@@ -272,3 +273,71 @@ def checkConfig(self, *args, **kwargs):
 
     def reconfigService(self, *args, **kwargs):
         return defer.succeed(None)
+
+
+class BuildbotServiceManager(AsyncMultiService, config.ConfiguredMixin,
+                             ReconfigurableServiceMixin, util.ComparableMixin):
```

Comment:
I like this idea -- can you fish it out into a PR that makes all of the existing service managers also use it?